packages/specs/schema/src/domain/JsonParameterStore.ts

```
@@ -114,19 +107,17 @@ export class JsonParameterStore extends JsonEntityStore {
     }
 
     this._type = this._type || Object;
+    this.parent.children.set(this.index!, this);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)